docs/en/04_Changelogs/4.0.0.md

```diff
@@ -92,6 +93,138 @@ For a full list of renamed classes, check the `.upgrade.yml` definitions in each
 
 The rename won't affect class-based permission codes or database table names.
 
+##### Use ::class whenever possible
```

Comment:
These upgrade steps are very low-priority. I suggest you put higher-priority important upgrade tasks at the top, and more "style guide recommendations" near the bottom of the list.